src/DataViews/FunctionsDataView.cpp

```diff
@@ -248,10 +247,15 @@ void FunctionsDataView::DoFilter() {
 void FunctionsDataView::AddFunctions(
     std::vector<const orbit_client_data::FunctionInfo*> functions) {
   functions_.insert(functions_.end(), functions.begin(), functions.end());
-  indices_.resize(functions_.size());
-  for (size_t i = 0; i < indices_.size(); ++i) {
-    indices_[i] = i;
-  }
+  OnDataChanged();
+}
+
+void FunctionsDataView::RemoveFunctionsOfModule(const std::string& module_path) {
```

Comment:
There is this code around `OrbitApp::RefreshUIAfterModuleReload` that currently calls `ClearFunctions` and then `AddFunctions` for all modules. In case you want to refactor that to use `RemoveFunctionsOfModule`, that would be great ;)

Reply:
Is there a bug I can link in the meantime?

Reply:
No, not that I am aware of

Reply:
Created and linked http://b/247069854